### PR TITLE
✨ Add City Edit Button to PlanPage

### LINKED ISSUE #25: As a User I want to type in the Citys in my overview
- [] Add Edit Button
- [] Add Function to change local Storage

### PR DESCRIPTION
**Pull request by m-wiemers**

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
LGTM! 🚀

File: components/planDetails/PlanDetails.tsx [hunk 1]
```diff
@@ -17,15 +22,45 @@ function PlanDetails({
...
+  inputChange,
+  placeHolder,
 }: PlanDetailProps) {
+  const [visible, setVisible] = useState(false);
+  function handleCityChange(label) {
+    if (visible === true) {
+      handleChange(label);
+    }
+    setVisible(!visible);
+  }
```
Comment (fre-ben) [inline components/planDetails/PlanDetails.tsx:35]: Nice! 💯

**Review by fre-ben — APPROVED** [review]
LGTM! 🦘

**Review by mewiemers — APPROVED** [review]
GJ!🦘

**Pull request merged**